webapp-django/crashstats/crashstats/static/crashstats/js/socorro/pending.js

```diff
@@ -1,47 +1,38 @@
+var Checker = (function() {
```

Comment:
This exposes ``Checker`` in the global namespace. If that's not what you deliberately intended, I guess it should be moved into a closure.